libimage/runtime.go

```diff
@@ -316,21 +316,32 @@ func (r *Runtime) LookupImage(name string, options *LookupImageOptions) (*Image,
 		return img, name, err
 	}
 
-	return r.lookupImageInDigestsAndRepoTags(name, options)
+	return nil, "", fmt.Errorf("%s: %w", name, storage.ErrImageUnknown)
+	return r.lookupImageInRepoTags(name, namedName, options)
 }
 
 // lookupImageInLocalStorage looks up the specified candidate for name in the
 // storage and checks whether it's matching the system context.
 func (r *Runtime) lookupImageInLocalStorage(name, candidate string, options *LookupImageOptions) (*Image, error) {
 	logrus.Debugf("Trying %q ...", candidate)
-	img, err := r.store.Image(candidate)
```

Comment:
@Luap99, I found that `store.Image` does _not_ resolve with digest inputs which is probably why the code had to do this digest dance in the first place.  In its new state, it works as expected.

I expect Buildah and Podman CI to pass at that point.

Reply:
Still some polishing needed in this PR, so just an FYI